database/migrations/2024_06_04_085042_add_daemon_sftp_alias_column_to_nodes.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('nodes', function (Blueprint $table) {
+            $table->text('daemon_sftp_alias')->nullable()->after('daemon_sftp');
```

Comment:
```suggestion
            $table->string('daemon_sftp_alias')->nullable()->after('daemon_sftp');
```

Reply:
I used `text` because the allocation alias also uses `text`. But `string` should be fine too.